### DISCUSSION TITLE
Be able to share a specific window and/or a desktop specfic region

### DISCUSSION THREAD

**Discussion by damianh** [mistweaverco/bananas#31] [Ideas]
Nice tool!

I don't always want to share my whole screen for privacy reasons. Also as an ultra-widescreen user, it's ofte better for viewers to be seeing a smaller region.

Would also help the experience for folks not using ultrawidescreen monitors with those that do.

**Comment by gorillamoe** [comment]
Leave that open for discussion, but it's basically in the name "Bananas Screen Sharing" and not "Bananas Window Sharing" or "Bananas Screen Area Sharing"

I understand the point though: Because of privacy reasons.

That's not a big deal for me, because I can mute all notifications on Linux with one click, but I don't know about Mac or Windows.

I'm also an ultra wide user, but I haven't had any issues with sharing my screen to 1080p folks, because you can zoom in and move the remote screen around as a participant.

I'm not totally against this, but to be fully transparent, I won't start working on that in the next releases, because there are things more important to me (and I hope to user users as well)

So let's discuss.

**Reply by damianh** [reply]
Understood. As an OSS maintainer I completely respect your postion to work on what interests/drives you.

For me in various web conference tools (zoom, meet etc) I only ever share a window, never a full screen. I occasionally use [RegionToShare](https://apps.microsoft.com/detail/9n4066w2r5q4?hl=en-us&gl=US) too.

Cool app. Thanks for sharing.

**Comment by gorillamoe** [comment]
This should work in 0.2.1.
